src/generated/resources/data/tropicraft/tags/items/ashen_masks.json

```diff
@@ -1,18 +1,18 @@
 {
```

Comment:
Changes to this file need to be discarded.

Reply:
done